reactor-core/src/main/java/reactor/util/retry/Retry.java

```diff
@@ -92,6 +114,16 @@
 		 */
 		Throwable failure();
 
+		/**
+		 * Return a read-only view of the user provided context, which may be used to store
+		 * objects to be reset/rollbacked before a retry.
+		 *
+		 * @return a read-only view of a user provided context.
+		 */
+		default ContextView retryContextView() {
```

Comment:
I agree on using `ContextView` suffix, as this is the part of the API that will be most visible to users and thus further clarifies that this is read only.